docs/central-install-digital-ocean.rst

```diff
@@ -90,16 +90,15 @@ Once you are in your server, you'll want to change your password so that people
 Changing Server Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. Make sure you are running Docker Engine v23.x and Docker Compose v2.16.x or greater.
+1. Make sure you are running Docker Engine v23.x and Docker Compose v2.16.x or greater.
```

Comment:
These should use dynamic numbering. If the dynamic numbering doesn't work there's something wrong with the indentation.